php/php.editor/src/org/netbeans/modules/php/editor/typinghooks/PhpCommentGenerator.java

```diff
@@ -114,6 +114,8 @@ private static void generateFunctionDoc(BaseDocument doc, int offset, int indent
 
         if (i.hasReturn) {
             generateDocEntry(doc, toAdd, "@return", indent, null, i.getReturnType()); // NOI18N
+        } else {
+            generateDocEntry(doc, toAdd, "@return", indent, null, Type.VOID); // NOI18N
```

Comment:
Nitpick: maybe simpler? 

```suggestion
            generateDocEntry(doc, toAdd, "@return", indent, null, i.hasReturn ? i.getReturnType() : Type.VOID); // NOI18N
```